google-cloud-spanner/src/main/java/com/google/cloud/spanner/Mutation.java

```diff
@@ -464,7 +494,32 @@ static void toProto(Iterable<Mutation> mutations, List<com.google.spanner.v1.Mut
     }
     // Flush last item.
     if (proto != null) {
+      com.google.spanner.v1.Mutation builtMutation = proto.build();
       out.add(proto.build());
+      if (checkIfInsertMutationWithLargeValue(builtMutation, largeInsertMutation)) {
+        largeInsertMutation = builtMutation;
+      }
+      if (!builtMutation.hasInsert()) {
+        allMutationsExcludingInsert.add(builtMutation);
+      }
```

Comment:
This code block is repeated a couple of times. Could we extract that to a separate method?

Reply:
I have extracted them to seperate methods. Please let me know if there are any other better ways.